app/components/dashboard/DcChart.vue

```
@@ -455,6 +451,12 @@
                 <label v-html="pnl.cond.title" for="stack_type_con"></label>
               </label>
 
+              <!-- FilterText -->
+              <a class="reset btn_reset" id="btn_reset_year" href="javascript:void(0);" style="display: none;">
```

ESLint

[error] 'style' should be on a new line.

(vue/max-attributes-per-line)